libs/auth/testing/src/factories/customer-registration.factory.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+
+import { DaffCustomerRegistration } from '@daffodil/auth';
+import * as faker from 'faker/locale/en_US';
+import { DaffModelFactory } from '@daffodil/core/testing';
+
+export class MockCustomerRegistration implements DaffCustomerRegistration {
+  email = faker.random.number(1000000).toString();
```

Comment:
Think this should probably be in an actual email format. Pretty sure you can use `faker.internet.email()`